reference/3.0/Microsoft.PowerShell.Core/About/about_Language_Modes.md

```diff
@@ -110,37 +110,35 @@ The features of ConstrainedLanguage mode are as follows:
   modules export run in the session.
 
 - In PowerShell Workflow, you can write and run script workflows (workflows
-  written in the PowerShell language). XAML-based workflows are not
-  supported and you cannot run XAML in a script workflow, such as by using
-  "Invoke-Expression -Language XAML". Also, workflows cannot call other
+  written in the PowerShell language). XAML-based workflows are not supported
+  and you cannot run XAML in a script workflow, such as by using
+  `Invoke-Expression -Language XAML`. Also, workflows cannot call other
   workflows, although nested workflows are permitted.
 
-- The Add-Type cmdlet can load signed assemblies, but it cannot load
-  arbitrary C# code or Win32 APIs.
+- The Add-Type cmdlet can load signed assemblies, but it cannot load arbitrary
+  C# code or Win32 APIs.
 
 - The New-Object cmdlet can be used only on allowed types (listed below).
 
-- Only allowed types (listed below) can be used in PowerShell. Other types
-  are not permitted.
+- Only allowed types (listed below) can be used in PowerShell. Other types are
+  not permitted.
 
-- Type conversion is permitted, but only when the result is an allowed
-  type.
+- Type conversion is permitted, but only when the result is an allowed type.
 
 - Cmdlet parameters that convert string input to types work only when the
   resulting type is an allowed type.
 
-- The ToString() method and the .NET methods of allowed types (listed
-  below) can be invoked. Other methods cannot be invoked.
+- The ToString() method and the .NET methods of allowed types (listed below)
```

Comment:
ToString() should be bold to follow the articles formatting.

Reply:
Fixed